.golangci.reference.yml

```diff
@@ -418,6 +418,14 @@ linters-settings:
     # Default: false
     suppress-err-assertion: true
 
+    # Suppress the wrong comparison assertion warning.
+    # Default: false
+    suppress-compare-assertion: true
+
+    # Don't trigger warnings for HaveLen(0)
+    # Default: false
+    allow-havelen-0: true
```

Comment:
I wonder if using a number inside a configuration is a good idea, I always apprehend this kind of element that the configuration parsers may not like.

Maybe `allow-havelen-zero` can work :thinking:

Reply:
Done.